app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def link_to_add_fields(name, f, model_class, association, cssClass)
```

Comment:
basically, this helper does the same thing [as](https://github.com/Jeezman/moniepoint/pull/1/files#diff-0a2d71f9054ceab017fca03fabd16083b4961523d74ead75029f35fff6721d2aR82-R84)
Only exception is that it instantiates a new record like we're doing with the `build` above and using that value to build a form template that is now serialized as string on the link.
You may inspect the `Add New Location` link to see the data attributes.